internal/reconcilers/entity_delete.go

```diff
@@ -58,8 +60,11 @@ func (r *Reconciler) handleEntityDeleteEvent(msg *message.Message) error {
 	l.Info().Msg("handling entity delete event")
 	// Remove the entry in the DB. There's no need to clean any webhook we created for this repository, as GitHub
 	// will automatically remove them when the repository is deleted.
-	if err := r.repos.DeleteByID(ctx, event.EntityID, event.ProjectID); err != nil {
-		return fmt.Errorf("error deleting repository from DB: %w", err)
+	err := r.repos.DeleteByID(ctx, event.EntityID, event.ProjectID)
+	if errors.Is(err, sql.ErrNoRows) {
+		return nil
```

Comment:
**nit:** This codepath only flows from webhook events, right? I would at least add a log message, I find it weird that we try to delete by ID something that does not exist anymore.

Reply:
I'll fix this in a follow-up where I ignore another such error